src/main/java/com/dataguise/saas/service/AgentService.java

```diff
@@ -0,0 +1,301 @@
+package com.dataguise.saas.service;
+
+
+import com.dataguise.saas.exception.BadGatewayException;
+import com.dataguise.saas.exception.BadRequestException;
+import com.dataguise.saas.util.Utility;
+import com.dg.saas.orch.client.DgSecureRestDriver;
+import com.dg.saas.orch.exception.DgDrvException;
+import com.dg.saas.orch.models.builders.AgentBuilder;
+import com.dg.saas.orch.models.structures.ErrorConstants;
+import com.dg.saas.orch.models.structures.IpRange;
+import io.swagger.client.model.*;
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ *
+ * The agent-management service
+ */
+@Service
+public class AgentService {
+
+    private static Logger logger = Logger.getLogger(AgentService.class);
+
+    @Autowired
+    private DgSecureRestDriver restDriver;
+
+
+    public Boolean addAgentStruct(List<DgAgentInfoStruct> dgAgentInfoStructList) throws BadRequestException,
+            BadGatewayException{
+        Boolean agentAdded = false;
+        int count =0;
+        for(DgAgentInfoStruct agentInfo : dgAgentInfoStructList){
+            addAgent(agentInfo);
+            count++;
+        }
+        if(count == dgAgentInfoStructList.size())
+            agentAdded = true;
+
+        return agentAdded;
+    }
+
+
+
+    /**
+     * Add a agent
+     */
+    public Integer addAgent(DgAgentInfoStruct agentInfoStruct)
+            throws BadRequestException,BadGatewayException {
+        /** Function to add multiple agents, ip ranges and group id mappings for detection agents. **/
+        Integer groupId = null;
+        Integer agentId = null;
+        Integer ipRangeId = null;
+
+        try {
+            validateAgent(agentInfoStruct);
+            logger.info("Updated Agent info struct: " + agentInfoStruct);
+        } catch (BadRequestException e) {
+            throw e;
+        }
+
+
+        try {
+            if(agentInfoStruct.getAgentType() == AgentBuilder.AgentType.S3Cloud.name() ||
+                    agentInfoStruct.getAgentType() == AgentBuilder.AgentType.UnixFiles.name() )
+            {
+                agentId = restDriver.configureAgent(agentInfoStruct);
+            }
```

Comment:
**Reporter**: Checkstyle    
    **Rule**: com.puppycrawl.tools.checkstyle.checks.blocks.RightCurlyCheck
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/service/AgentService.java L73
    
    &#39;}&#39; at column 13 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/finally).
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a38421601>*